### ISSUE TITLE
Provide "Preload specific font file/s" as Partial

### ISSUE THREAD

**Issue by stephanschroeter** [porsche-design-system/porsche-design-system#1119]
Scope
---
Describe clearly the scope of the task.

Out of Scope
---
List topics that are not part of this task.

Notes  
---
* Use as interface: `getFontPreload({ weight: 'thin' | 'regular' | 'semibold' | 'bold' | 'all', subset: 'latin' | 'cyril' | 'greek', cdn: 'auto' | 'cn' })`
* Use woff2 only
* Keep placeholder comment while/after replacing it with partial content (for Angular/VanillaJS integration)

### Relationship options:

`dns-prefetch`:
- Browser resolves dns to a specific resource ahead-in-time

`preconnect`: 
- Like dns prefetch but with TCP handshake and optional TLS negotiation

`prefetch`:
- browser can prefetch resource, low prio though
- https://caniuse.com/link-rel-prefetch

`subresource`:
- (was used to fetch something before prefetch, is now removed in chrome)

`preload`: 
- browser forced to fetch before document onload
- not working in older browsers https://caniuse.com/link-rel-preload 
- must use "as" parameter to specify resource e.g 
`<link rel="preload" href="someFont.woff2" as="font">`

Both prefetch and preload strategies cache the resource as soon as possible

`prerender`: 
- Loads (pre-renders) specified webpage in the background, heavy on bandwidth, should not prerender if not absolutely sure, that the link to the page is going to get clicked by the user
-(e.g. Google uses that to load entire pages in the background before user clicks on a link)

TLDR:
For the fonts, use `preload` if browser support is available, otherwise use `prefetch`.

Acceptance criteria
---
- [x] Criteria

Subtasks
---
- [x] Task
